src/main/java/uk/gov/digital/ho/hocs/search/client/elasticsearchclient/ElasticSearchMultipleClient.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.digital.ho.hocs.search.client.elasticsearchclient;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.elasticsearch.client.RestHighLevelClient;
+import org.elasticsearch.index.query.BoolQueryBuilder;
+import uk.gov.digital.ho.hocs.search.domain.model.CaseData;
+
+import java.util.Set;
+import java.util.UUID;
+
+public class ElasticSearchMultipleClient extends BaseElasticSearchClient {
+
+    public ElasticSearchMultipleClient(ObjectMapper objectMapper, RestHighLevelClient client, String aliasPrefix) {
+        super(objectMapper, client, aliasPrefix);
+    }
+
+    @Override
+    public CaseData findById(UUID uuid) {
+        return findById(getReadAlias(), uuid);
```

Comment:
This doesn't work because you cannot do a document find on an alias that is linked to multiple indices. 

Options:
1. Supply a case uuid -> case type lookup that enabled you to use a specific index. We'd have to change to a one read alias per index.
2. Change all uses to ensure they pass the case type across rather than rely on a lookup config file
3. Search all of the indices instead using a limited query - this does not provide realtime results

>